src/wallet/rpcwallet.cpp

```diff
@@ -1359,6 +1359,27 @@ static void ListTransactions(interfaces::Chain::Lock& locked_chain, CWallet* con
     }
 }
 
+static const std::string TransactionDescriptionString()
+{
+    return "    \"confirmations\": n,                        (numeric) The number of confirmations for the transaction. Negative confirmations indicate the\n"
+           "                                                       transaction conflicts with the block chain.\n"
+           "    \"generated\": xxx,                          (bool) Only returns true if transaction only input is a coinbase one, otherwise false.\n"
+           "    \"trusted\": xxx,                            (bool) Whether we consider the outputs of this unconfirmed transaction safe to spend.\n"
```

Comment:
Could you update this to be `(bool, optional)` and address [this comment](https://github.com/bitcoin/bitcoin/pull/16951#discussion_r327494296):

> if this is optional, I think the help text should also mention when the value is present or not.

Reply:
Rewrote comment but not sure about optional, it seems to be reserved when a optional feature is involved like HD-wallet, which is not the case with IsTrusted